hushline/routes.py

```diff
@@ -518,10 +526,15 @@ def logout() -> Response:
     @app.route("/directory")
     def directory() -> Response | str:
         logged_in = "user_id" in session
+        # Provide a default intro text if DIRECTORY_INTRO is not set
+        intro_text_setting = OrganizationSetting.fetch_one(OrganizationSetting.DIRECTORY_INTRO)
+        intro_text = intro_text_setting.value if intro_text_setting else "Welcome to the directory."
```

Comment:
this if/else will not handle the case where there is empty/null intro text. we likely want:

```python
if  (intro_text_setting := OrganizationSetting.fetch_one(OrganizationSetting.DIRECTORY_INTRO)) \
    and intro_text_setting.value:
    intro_text = intro_text_setting.value
else:
    intro_text = "Welcome to the directory."
```

Reply:
11d994c